src/lua/tests.rs

```diff
@@ -0,0 +1,31 @@
+//! Lua thread test.
+//! Spawns a lua thread!
+
+use std::time::Duration;
+use std::thread;
+
+use super::*;
+
+#[test]
+fn big_lua_test() {
+    println!("Testing lua thread...");
+    super::init();
+    thread::sleep(Duration::from_secs(5));
```

Comment:
It would be nice to have that as a testing method, if you want we could expose it but it should definitely be as another option since if all requests did that then we would have the threads never die if something goes wrong